forth_src/base.fs

```diff
@@ -182,17 +182,21 @@ does> dup @ to here
 
 marker ---modules---
 
+
+.( labels..) include labels
+.( doloop..) include doloop
+.( sys..) include sys
+.( debug..) include debug
 .( wordlist..) include wordlist
```

Comment:
As `find-name` is no longer in the core, it needs more facilities before loading.